samples/server/petstore/python-flask-python2/openapi_server/test/test_spec_parsing.py

```diff
@@ -0,0 +1,15 @@
+# coding: utf-8
+
+import connexion
+import logging
+from openapi_server.encoder import JSONEncoder
+from unittest import TestCase
+
+class TestSpecParsing(TestCase):
+    def test(self):
+        with self.assertRaises(TypeError):
+            logging.getLogger('connexion.operation').setLevel('ERROR')
+            app = connexion.App(__name__, specification_dir='../openapi/')
+            app.app.json_encoder = JSONEncoder
+            app.add_api('broken_openapi.yaml', pythonic_params=True)
```

Comment:
Can you trim that spec down to just the part that breaks functionality?
Can you add a comment in the yaml at that location?
Which code is failing to handle the spec or the datetime correctly?
Is it us or connexion?
- If it's connection it may make more sense to open an issue with that repo